examples/substreams-powered-subgraph/proto/example.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
```

Comment:
we should link some intro protobuf docs here? https://protobuf.dev/programming-guides/proto3/#simple